content/en/docs/reference/using-api/deprecation-guide.md

```diff
@@ -20,6 +20,18 @@ deprecated API versions to newer and more stable API versions.
 
 ## Removed APIs by release
 
+### v1.27
+
+The **v1.27** release will stop serving the following deprecated API versions:
```

Comment:
It is deprecated since 1.24, i.e. will be served in 1.24, 1.25, 1.26, (the required three releases) but not 1.27.

/lgtm

Reply:
Thanks @pohly for serving as a technical reviewer for this!